pylzy/tests/test_data/cassettes/env/explorer/test_classify/test_classify_local_distribution.yaml

```diff
@@ -0,0 +1,54 @@
+interactions:
+- request:
+    body: null
+    headers:
+      Accept-Encoding:
+      - gzip, deflate
+      Connection:
+      - keep-alive
+      User-Agent:
+      - pylzy 1.15.0 (https://lzy.ai)
```

Comment:
Fixed version here
Maybe not add version to User-Agent?

Reply:
It doesn't matter and will not affect tests after version changes.
Or it insults your a sense of beauty?